/**
 * @file /qlistener/listener.hpp
 *
 * @brief Ros communication central!
 *
 * @date February 2011
 **/
/*****************************************************************************
** Ifdefs
*****************************************************************************/

#ifndef LISTENER_NODE_HPP_
#define LISTENER_NODE_HPP_

/*****************************************************************************
** Includes
*****************************************************************************/

#include <iostream>
#include <fstream>
#include <sstream>
#include <std_msgs/String.h>
#include "walkinggait/Update.h"
#include "walkinggait/Serial.h"
#include <std_msgs/String.h>
#include <std_msgs/Int8.h>
#include <string>
#include "../classInfo/classinfo.hpp"
#include "../common/Strategy.hpp"

#include <ros/ros.h>


#define SIZE 100
using namespace std;

#endif /* LISTENER_NODE_HPP_ */
